### DISCUSSION TITLE
Cookbook website from Project Pythia

### DISCUSSION THREAD

**Discussion by samsrabin** [NCAR/ctsm_python_gallery#45] [Ideas]
https://cookbooks.projectpythia.org/

Giving me lots of good ideas for a more user-friendly version of this repo!

**Comment by katiedagon** [comment]
Yes, it would be awesome to model this after Pythia cookbooks! 
FYI, there will be a Pythia "cook-off" in June: https://projectpythia.org/pythia-cookoff-2023/README.html
